site/en/tutorials/estimator/premade.ipynb

```diff
@@ -230,8 +230,8 @@
         "`tf.estimator`\n",
         "(for example, `LinearRegressor`) to implement common ML algorithms. Beyond\n",
         "those, you may write your own\n",
-        "[custom Estimators](https://www.tensorflow.org/guide/custom_estimators).\n",
-        "We recommend using pre-made Estimators when just getting started.\n",
+        "[custom Estimators](https://www.tensorflow.org/guide/estimator#custom_estimators).\n",
+        "It is recommend using pre-made Estimators when just getting started.\n",
```

Comment:
```suggestion
        "It is recommended using pre-made Estimators when just getting started.\n",
```

Reply:
Yep I'm on it, Haven't finished my commit yet, Will inform you when I'm done :)